mantidimaging/core/rotation/polyfit_correlation.py

```diff
@@ -24,9 +24,29 @@ def do_calculate_correlation_err(store, search_index: int, p0_and_180, image_wid
     store[:] = np.square(np.roll(p0_and_180[0], search_index, axis=1) - p0_and_180[1]).sum(axis=1) / image_width
 
 
-def find_center(images: ImageStack, progress: Progress) -> tuple[ScalarCoR, Degrees]:
-    if images is None or images.proj180deg is None:
-        raise ValueError("images and images.proj180deg cannot be None")
+def find_center(images: ImageStack,
+                progress: Progress,
+                use_projections: tuple[int, int] | None = None) -> tuple[ScalarCoR, Degrees]:
+
+    if images is None:
+        raise ValueError("images cannot be None")
+
+    proj180 = getattr(images, "proj180deg", None)
+    if use_projections is not None:
+        start_idx, end_idx = use_projections
+        proj_a = images.projection(start_idx)
+        proj_b = np.fliplr(images.projection(end_idx))
+        LOG.info(f"Using projections {start_idx} and {end_idx} for correlation")
+    elif proj180 is not None:
+        proj_a = images.projection(0)
+        proj_b = np.fliplr(proj180.data[0])
+        LOG.info("Using flipped images.proj180deg for correlation (legacy behaviour)")
+    else:
+        start_idx = 0
+        end_idx = (images.num_projections // 2) % images.num_projections
+        proj_a = images.projection(start_idx)
+        proj_b = np.fliplr(images.projection(end_idx))
+        LOG.info(f"No proj180deg found; using projections {start_idx} and {end_idx}")
```

Comment:
Let raise an exemption in this third case. The caller has not given a proj180 or `use_projections`.